### PR TITLE
Buyer's Guide: Product page links style & behaviour

### LINKED ISSUE #2094: Product page: link styling and placement updates
Based on user testing & feedback at MozFest https://github.com/mozilla/foundation.mozilla.org/issues/2072 and the [comp](https://redpen.io/p/qz6bdb2811e058ffb1).

### Update location of company page link
 - [ ] Move URL link to the product company and price (URL link is currently in the cms and linked to the product title)
- [ ] Add underline to text
- [ ] Open link in new tab
- [ ] Hover state #3302FF
image

### Updates to Copy Link
- [ ] Update `copy link` text to: `copy & share link`
- [ ] Add icon [icon-product-link.svg.zip](https://github.com/mozilla/foundation.mozilla.org/files/2586827/icon-product-link.svg.zip)
- [ ] Font style: same as Company name & price with hover state #3302FF
- [ ] Move link up so it's inline with company & price
- [ ] Let users know they have successfully copied the link url 
  - Briefly (3 seconds) update copy to `Link copied!` 
  - #3302FF

screen shot 2018-11-07 at 11 21 56 am

screen shot 2018-11-07 at 11 21 44 am

### PR DESCRIPTION
**Pull request by mmmavis**
Closes #2094

The 3 seconds delay after user clicks on "copy & share link" feels too long. The copying task completes almost simultaneously... I wonder if we really need to add long delay (or any delay at all)?

### CODE REVIEW

**Comment by kristinashu** [comment]
Oh sorry, the 3 seconds is for showing the "link copied!" text. So user clicks "copy & share link", then they see"link copied!" right away, then it goes back to  "copy & share link" after 3 seconds.

**Comment by kristinashu** [comment]
The font looks thicker than it did before, can you please keep it the same weight as on prod?

File: source/js/buyers-guide/bg-main.js [hunk 1]
```diff
@@ -82,6 +82,7 @@ let main = {
             });
           }
 
+          let target = event.target;
```
Comment (Pomax) [inline source/js/buyers-guide/bg-main.js]: if we only use `target` in that try block, let's move it there.

File: source/js/buyers-guide/bg-main.js [hunk 2]
```diff
@@ -126,6 +127,13 @@ let main = {
 
           try {
             document.execCommand(`copy`);
+
+            if (target.dataset.successText) {
```
Comment (Pomax) [inline source/js/buyers-guide/bg-main.js]:
```suggestion
            if (target.dataset && target.dataset.successText) {
```

**Pull request merged**